docs/content/templates/functions.md

```diff
@@ -460,6 +460,22 @@ e.g.
 
 * `{{ int "123" }}` → 123
 
+### lang.NumFmt
+
+`NumFmt` formats a number with a given precision using the requested *decimal*,
+*grouping*, and *negative* characters.  The function signature is
+`NumFmt <decimal> <grouping> <negative> <precision> <number>`.
+
+Note that numbers are rounded up at 5 or greater.
+So, with precision set to 0, 1.5 becomes `2`, and 1.4 becomes `1`.
+
+```
+{{ lang.NumFmt "," "." "-" 2 12345.6789 }} → 12.345,68
```

Comment:
But I think we should just drop the locale debate, and just pack the three arguments into one string. It should be possible to do some magic locale stuff later if needed.